contracts/contracts/LpWhitelist.sol

```diff
@@ -29,6 +29,12 @@ contract LpWhitelist is Ownable {
         _;
     }
 
+    /// Modifier to check whether a given address is whitelisted
+    modifier whitelistedAddress(address addressToCheck) {
+        require(allowedLps[addressToCheck], "Given address not whitelisted");
+        _;
+    }
```

Comment:
I would propose to merge this with the `onlyWhitelist` modifier and so that we have just one modifier, e.g.
```
modifier onlyWhitelisted(address ...)
```
I would also shorten the error message to `"Not whitelisted"`.

Thinking further about the naming here, we have `allowedLps` and `whitelist` -- it would be nice to have some consistency there, e.g.

```
modifier onlyAllowed(address ...)
```
with the message then simply e.g. ` "Not allowed"`.